ui/readme.fr.md

```diff
@@ -96,6 +96,13 @@ utilisez simplement cette syntaxe spéciale : préfixez votre requête par un **
 Par exemple, **+chemise femme**.  
 Ces recherches spéciales renvoient soit un seul groupe si une variante correspond exactement à la requête saisie, soit un résultat vide s'il n'existe aucun groupe correspondant à cette requête.
 
+# Redirects
```

Comment:
Please use French headline. Use the same wording as the UI in the left-hand side menu.  Add `{redirects}` at the end.